modules/react/combobox/lib/ComboboxMenuItem.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import {
+  composeHooks,
+  createElemPropsHook,
+  createSubcomponent,
+} from '@workday/canvas-kit-react/common';
+import {StyledMenuItem, useMenuModel} from '@workday/canvas-kit-react/menu';
+import {
+  useListItemAllowChildStrings,
+  useListItemRegister,
+} from '@workday/canvas-kit-react/collection';
+import {OverflowTooltip} from '@workday/canvas-kit-react/tooltip';
+
+import {useComboboxModel} from './useComboboxModel';
+
+export interface ComboboxMenuItemProps {
+  /**
+   * Children of the Combobox. Should contain a `<Combobox.Target>`, a `<Combobox.Content>`
+   */
+  children: React.ReactNode;
+}
+
+export const useComboboxMenuItem = composeHooks(
+  createElemPropsHook(useMenuModel)((model, _, elemProps: {'data-id'?: string} = {}) => {
+    const id = elemProps['data-id'] || '';
+    const onMouseDown = (event: React.MouseEvent<HTMLElement>) => {
+      if (
+        model.state.nonInteractiveIds.includes(id) ||
+        event.currentTarget.hasAttribute('aria-disabled')
```

Comment:
should this account is something also adds `disabled` ?

Reply:
`disabled` is not valid on an item and if it is added, no events will be fired on it.

Reply:
I should clarify:

```jsx
<button disabled onMouseDown={() => console.log('The browser will prevent this event from ever firing')} />
```